File: php/testcode/benchmark_test_00200.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00200(BenchmarkRequest $req): BenchmarkResponse {
    $dir = $req->param('dir');
    pcntl_exec('/usr/bin/ls', [escapeshellarg($dir)]);
    return BenchmarkResponse::ok('executed');
}

File: rust/testcode/benchmark_test_00200.rs
use std::collections::HashMap;

pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let mut m: HashMap<&str, String> = HashMap::new();
    m.insert("user_url", req.param("url"));
    m.insert("safe_url", "https://api.example.com/data".to_string());

    let url = m.get("safe_url").unwrap();

    let _resp = simulated_reqwest_get(url);

    super::shared::BenchmarkResponse::ok("Fetched")
}

fn simulated_reqwest_get(url: &str) -> String {
    format!("Response from {}", url)
}
